### DISCUSSION TITLE
Kingsmith K15

### DISCUSSION THREAD

**Discussion by cagnulein** [cagnulein/qdomyos-zwift#460] [Q&A]
@d3m3vilurr hi! how are you? an user (not tech savvy) needs help to collect the wifi protocol of the kingsmith K15. Are you able to help him in some way?

**Comment by cagnulein** [comment]
the app that the user is currently using for the treadmill is the Mi Home

**Reply by d3m3vilurr** [reply]
It would have different protocol.
maybe home automation community have a protocol information using wifi + mi home (and it needs to extract auth token from mi home app)
idk right example but related issues; https://github.com/rytilahti/python-miio/issues/797

**Reply by cagnulein** [reply]
wow thanks!
